inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,213 @@
+						</a>
+					<# } #>
+				</div>
+			</script>
+			<?php
+		}
+
+		/**
+		 * Guided tour steps.
+		 *
+		 * @since 0.8.7
+		 */
+		public function guided_tour_steps() {
+			$steps = array();
+
+			$steps[] = array(
+				'title'       => __( 'Customize Your Store', 'storefront' ),
+				'message'     => __( 'It looks like your current theme isn\'t ready for shop features yet - your shop pages might look a little funny.</p><p>We suggest switching themes to <b>Storefront</b> which will bring out the best in your shop. Don\'t worry, if you try Storefront now, it won\'t be activated until you save your changes in the Customizer', 'storefront' ),
+				'buttonText'  => __( 'I\'ll keep my current theme', 'storefront' ),
+				'section'     => '#customize-info',
+				'className'   => 'sf-button-secondary',
+				'altStep'     => array(
+					'buttonText'  => __( 'I\'ll try Storefront!', 'storefront' ),
+					'action'      => 'expandThemes',
+					'message'     => __( 'Click the thumbnail to get started with Storefront', 'storefront' ),
+					'section'     => '#customize-control-theme_storefront .theme-screenshot',
+				)
+			);
+
+			// Determine what the next step should be
+			$needsLogo = ! has_custom_logo() && current_theme_supports( 'custom-logo' );
+			$logoBullet = $needsLogo ? __( '<li>Add your logo</li>', 'storefront' ) : '';
+
+			$steps[] = array(
+				'message'     => __( 'Okay! Remember you can switch themes at any time.</p><p>To get your store looking great, let\'s run through some common tasks:</p><ul>' . $logoBullet . '<li>Add Shop pages to your menus</li><li>Set your shop page as the homepage</li></ul><p>', 'storefront' ),
+				'buttonText' => $needsLogo ? __( 'Add a logo', 'storefront' ) : __( 'Add menu items', 'storefront' ),
+				'section'     => '#accordion-section-themes',
+				'altStep'    => array(
+					'buttonText' => __( 'I\'ll figure it out for myself', 'storefront' ),
+					'className'   => 'sf-button-secondary',
+					'action'      => 'exit',
+				)
+			);
+
+			if ( $needsLogo ) {
+				$steps[] = array(
+					'title'       => __( 'Add your logo', 'storefront' ),
+					'action'      => 'addLogo',
+					'message'     => __( 'Click the \'Select Logo\' button to upload your logo. After you upload your logo, click next to update your menus.', 'storefront' ),
+					'buttonText'  => __( 'Next', 'storefront' ),
+					'section'     => 'title_tagline',
+				);
+			}
+
+			$steps[] = array(
+				'message'      => __( 'Choose a menu to add shop pages to', 'storefront' ),
+				'section'      => '#sub-accordion-panel-nav_menus',
+				'panel'        => 'nav_menus',
+				'panelSection' => '.control-section-nav_menu',
+				'action'       => 'updateMenus',
+				'showSkip'     => 'true',
+				'childSteps'   => array(
+					array(
+						'message'    => __( 'Here are the items currently added to your menu. Click the "Add Items" button.', 'storefront' ),
+						'section'    => '.add-new-menu-item',
```

Comment:
When you have more than one menu, this doesn't always point to the correct element:
![misaligned](https://user-images.githubusercontent.com/541093/31023210-238a39f4-a509-11e7-9437-ae041374fcd7.png)

If you change the section element to `'.control-section-nav_menu.open .add-new-menu-item'`, it catches the right thing.